modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -872,6 +872,65 @@ def attempt_convert_item(input_value, valid_classes, path_to_item,
     return input_value
 
 
+def is_type_nullable(input_type):
+    """
+    Returns true if None is an allowed value for the specified input_type.
+
+    A type is nullable if at least one of the following conditions is true:
+    1. The OAS 'nullable' attribute has been specified,
+    1. The type is the 'null' type,
+    1. The type is a anyOf/oneOf composed schema, and a child schema is
+       the 'null' type.
+    Args:
+        input_type (type): the class of the input_value that we are
+            checking
+    Returns:
+        bool
+    """
+    if input_type is none_type:
+        return True
+    # TODO: how do we handle the 'nullable: true' scenario?
+    if issubclass(input_type, ModelComposed):
+        # If oneOf/anyOf, check if the 'null' type is one of the allowed types.            
+        for t in input_type._composed_schemas.get('oneOf', ()):
+            if t is none_type:
+                return True
+        for t in input_type._composed_schemas.get('anyOf', ()):
+            if t is none_type:
+                return True
+    return False
+
+
+def is_valid_type(input_class_simple, valid_classes):
+    """
+    Args:
+        input_class_simple (class): the class of the input_value that we are
+            checking
+        valid_classes (tuple): the valid classes that the current item
+            should be
+    Returns:
+        bool
+    """
+    valid_type = input_class_simple in valid_classes
+    if not valid_type and (
+            issubclass(input_class_simple, OpenApiModel) or
+            input_class_simple is none_type):
+        for valid_class in valid_classes:
+            if not valid_class.discriminator:
+                continue
+            if input_class_simple is none_type and is_type_nullable(valid_class):
```

Comment:
@spacether , I think this test is needed to handle the scenario when 1) the input value is None, AND 2) the oneOf/anyOf schema has a child schema which is the 'null' type. Is there a better way to handle this scenario?

Reply:
In my opinion this is the right way to test for this use case.
We can have a schema with:
oneOf:
 - null
 - integer
 - string
If we have this case with no discriminator, then we would never return True, because we continued on line 920
How about moving the none check to go before the discriminator continue block?

Reply:
done